app/views/admin/questions/index.html.slim

```diff
@@ -12,13 +12,13 @@
   table.table.table-bordered.table-hover
     tr.active
       th width="5%"
-        h4 Id
-      th width="15%"
+        h4 id
+      th width="10%"
         h4 Topic
-      th width="40%"
+      th width="50%"
         h4 Question
-      th width="30%"
-        h4 Answer
-      th colspan="3" width="10%"
+      th width="15%"
```

Comment:
We decided that answer field is unnecessary in table, but we can see answer when we click on question.